modules/govuk/manifests/apps/govuk_crawler_worker.pp

```diff
@@ -103,11 +111,13 @@
     }
 
     govuk::app { 'govuk_crawler_worker':
-      app_type           => 'bare',
-      log_format_is_json => true,
-      port               => $port,
-      command            => './govuk_crawler_worker -json',
-      health_check_path  => '/healthcheck',
+      app_type               => 'bare',
+      log_format_is_json     => true,
+      port                   => $port,
+      command                => './govuk_crawler_worker -json',
+      health_check_path      => '/healthcheck',
+      nagios_memory_warning  => $nagios_memory_warning,
+      nagios_memory_critical => $nagios_memory_critical,
```

Comment:
Unless you actually need different values in different environments, I think it's OK to put the actual values here, rather than in the hieradata.

There's nothing wrong with having them in the hieradata though.